lxl-web/src/lib/utils/LxlQueryHighlight.ts

```diff
@@ -0,0 +1,18 @@
+import { syntaxHighlighting } from '@codemirror/language';
+import { customTags as t } from 'codemirror-lang-lxlquery';
+import { tagHighlighter } from '@lezer/highlight';
+
+const highlighter = tagHighlighter([
+  { tag: t.Qualifier, class: 'qualifier' },
+  { tag: t.QualifierKey, class: 'qualifierkey' },
+  { tag: t.QualifierValue, class: 'qualifiervalue' },
+  { tag: t.EqualOperator, class: 'equaloperator' },
+  { tag: t.CompareOperator, class: 'compareoperator' },
+  { tag: t.BooleanOperator, class: 'booleanoperator' },
+  { tag: t.BooleanQuery, class: 'booleanquery' },
+  { tag: t.Wildcard, class: 'wildcard' }
```

Comment:
I think some kind of separation between the words is good for legibility. Sorry for nitpicking!

It's up for discussion but I have a slight preference for kebab-case in CSS classes (e.g. as on [MDN](https://developer.mozilla.org/)) – mainly to separate it from the default camelCase convention in JavaScript.

Reply:
> I have a slight preference for kebab-case in CSS classes

Mee too, somehow it got lost here!